### PR TITLE
docs: add cli tool section

### PR DESCRIPTION
**Pull request by 96-38**
## Improvements
- Added a CLI Tool section to the Translations page of the documentation.
I created with reference to the Config Editor page. However, I am not a native English speaker, so please let me know if there are any sentences that are grammatically incorrect.
CleanShot 2022-10-30 at 10 07 16@2x

### CODE REVIEW

File: docs/content/pages/tutorials/translation.md [hunk 1]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
     }
 }
 ```
+
+# CLI Tool
+
+Are you tired of manually translating JSON? Do you want an automatic translator? Well then the [nh-translation-helper](https://www.npmjs.com/package/nh-translation-helper) may be for you!
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:38]:
```suggestion
Are you tired of manually translating JSON? Do you want an automatic translator? Well then the [nh-translation-helper](https://www.npmjs.com/package/nh-translation-helper){ target="_blank" } may be for you!
```

File: docs/content/pages/tutorials/translation.md [hunk 2]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
...
+This tool has the following features:
+
+- Extract text from XML files and create english.json as the translation source.
+- Translate english.json to create a json file for another language.
+
+This section outlines how to install and use the nh-translation-helper.
+
+## Installation
+
+To get started, head over to the [repo for the tool](https://github.com/96-38/nh-translation-helper) and prepare the requirements:
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:49]:
```suggestion
To get started, head over to the [repo for the tool](https://github.com/96-38/nh-translation-helper){ target="_blank" } and prepare the requirements:
```

File: docs/content/pages/tutorials/translation.md [hunk 3]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
...
+
+- Install [Node.js](https://nodejs.org/) >= 12.0.0
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:51]:
```suggestion
- Install [Node.js](https://nodejs.org/){ target="_blank" } >= 12.0.0
```

File: docs/content/pages/tutorials/translation.md [hunk 4]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
...
+    - Install the LTS version.
+- Get [DeepL API](https://www.deepl.com/docs-api) Key (Free or Pro)
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:53]:
```suggestion
- Get [DeepL API](https://www.deepl.com/docs-api){ target="_blank" } Key (Free or Pro)
```

File: docs/content/pages/tutorials/translation.md [hunk 5]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
...
+    - Sign up [here](https://www.deepl.com/pro#developer)
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:54]:
```suggestion
    - Sign up [here](https://www.deepl.com/pro#developer){ target="_blank" }
```

File: docs/content/pages/tutorials/translation.md [hunk 6]
```diff
@@ -32,3 +32,69 @@ Here's an example, for `russian.json`:
     }
 }
 ```
+
+# CLI Tool
```
Comment (Bwc9876) [inline docs/content/pages/tutorials/translation.md:36]: Markdown best practice dictates that there shouldn't be more than one top-level heading, you may want to make this an `h2` and increment all the levels of the below headings by one.

**Review by Bwc9876 — CHANGES_REQUESTED** [review]
Looks good! Just a few UX things and some nit-picks.
- Change the links to use `{ target="_blank" }`, so they open in new tabs
- Make it so the CLI Tool heading is a second-level one and increment all the ones below it by one

File: docs/content/pages/tutorials/translation.md (hunk 6 above)
Comment (96-38) [reply docs/content/pages/tutorials/translation.md:36]: Actually, I had my own concerns about that.
When I viewed the Ship log page, I saw that there were multiple top-level headings, so I did the same.
However, I will change it if it looks like it should be fixed.
Thanks for the review ::)

**Review by Bwc9876 — APPROVED** [review]
LGTM!

**Pull request merged**